.github/workflows/update-version.yml

```diff
@@ -0,0 +1,37 @@
+name: Update Version
+on:
+  workflow_dispatch:
+    inputs:
+      version: 
+        description: 'New version to use for the Places extension. Example: 3.0.0'
+        required: true
+
+      core-dependency:
+        description: '[Optional] Update Core dependency in pom.xml. Example: 3.0.0'
+        required: false
+
+       edge-dependency:
+        description: '[Optional] Update Places dependency in pom.xml. Example: 3.0.0'
```

Comment:
`Update Places dependency` -> `Update Edge dependency`

Reply:
Places doesn't have a hard dependency on Edge..
I will remove this part